src/navigation-bar/index.ts

```diff
@@ -87,12 +91,12 @@ export function NavigationBar() {
     setCurrentFolder(rootFolder)
   }
 
-  const goUp = h("button", "⬆️")
+  const goUp = h("button", "🡱")
   const breadcrumb = h("ul", { className: "breadcrumb" })
-  const newFolderBtn = h("button", "New Folder")
-  const newFileBtn = h("button", "New File")
-  const renameNodeBtn = h("button", { disabled: true }, "Rename")
-  const deleteNodeBtn = h("button", { disabled: true }, "Delete")
+  const newFolderBtn = h("button", h("img", { src: folderImage }), h("span", "New"), h("br"), h("span", "Folder"))
```

Comment:
This suggests to me that we might need to create s new component that creates control bar buttons. But we can leave it for another pr, so as to not kill your flow

Reply:
For now give the button a class of `control-button` and style that instead of ‘... > button`